src/core/views.py

```diff
@@ -417,6 +418,13 @@ def edit_profile(request):
             request.journal
         ).value
 
+    if user.staffgroupmember_set.first():
```

Comment:
I think this works for our current use case but if we ever wanted to have one profile split across two groups there would be no way to edit the second entry. I don't think a change is needed now as it will slow down development but we should consider opening a ticket to convert this into a formset in future?

https://docs.djangoproject.com/en/3.2/topics/forms/formsets/